domain/entity/shift_position.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from sqlalchemy import Column, String, DateTime, ForeignKey, Integer, Enum
+from sqlalchemy.orm import relationship
+
+from domain.base import Base
+
+
+class ShiftPosition(Base):
+    __tablename__ = 'position'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    shift_id = Column(Integer, ForeignKey('shift.id'), nullable=False)
```

Comment:
This is causing a problem with the unit tests, I am going to assume you mean `shift_request`?